python/postprocessing/analysis/higgs/vhbb/VHbbProducer.py

```diff
@@ -55,42 +55,22 @@ def analyze(self, event):
             Vtype = 4
             if event.__getattr__("MET_pt") < 150:
                 Vtype = -1
-       
         self.out.fillBranch("Vtype",Vtype)
 
-        self.out.fillBranch("Jet_lepFilter",self.cleanJets(jets,electrons,muons))
+        ## filter jets that overlap with any of the selected leptons
+        allLeptons = zElectrons[:]
+        allLeptons.extend(zMuons)
+        allLeptons.extend(wElectrons)
+        allLeptons.extend(wMuons)
+        jetFilterFlags = [True]*len(jets)
+        for lepton in allLeptons:
+            jetInd = lepton.jetIdx
+            if jetInd >= 0:
+                jetFilterFlags[jetInd] = False
```

Comment:
is it possible/better to do something like jets[i].jetFilter = False here? I tried this but it didn't work, but maybe I was just missing something.

Reply:
mhh it should work, perhaps you can change
        jets = Collection(event, "Jet")
into
        jets = list(Collection(event, "Jet"))